api/app/tests/fba/test_fba_error.py

```diff
@@ -96,71 +114,117 @@ def given_input(elevation: float,  # pylint: disable=too-many-arguments, invalid
 
     return {
         'python': python_fba,
-        'java': java_fbp
+        'java': java_fbp,
+        'fuel_type': fuel_type
     }
 
 
-def relative_error(actual: float, expected: float, precision: int = 2):
+def relative_error(metric: str, actual: float, expected: float, precision: int = 2):
     """ calculate the relative error between two values - default to precision of 2"""
     actual = round(actual, precision)
     expected = round(expected, precision)
+    if actual == expected:
+        return 0
+    if expected == 0:
+        raise RelativeErrorException(
+            f'unable to calculate relative error for {metric}; actual:{actual};expected:{expected}')
     return abs((actual-expected)/expected)
```

Comment:
Is it possible that we're dividing by zero here?

Reply:
I guess it doesn't matter since this is a test

Reply:
Check for 0 is happening above.